programmers/난이도별/level01.콜라츠_추측/HyeonJeong.py

```diff
@@ -0,0 +1,8 @@
+def solution(num):
+    tries = 0
+    while num != 1:
+        num = num // 2 if num % 2 == 0 else num * 3 + 1
+        tries += 1
+        if tries == 500:
```

Comment:
```suggestion
        if tries >= 500:
```
보통 혹시 모를 일을 위해서 이런식으로 많이 사용합니다!